app/src/main/java/org/mozilla/focus/home/pocket/PocketVideoFragment.kt

```diff
@@ -103,10 +108,19 @@ class PocketVideoFragment : Fragment() {
 
 private class PocketVideoAdapter(
     context: Context,
-    private val pocketVideos: List<PocketVideo>,
     private val fragmentManager: FragmentManager
 ) : RecyclerView.Adapter<PocketVideoViewHolder>() {
 
+    private val pocketVideos = mutableListOf<PocketVideo>()
+
+    inner class VideoObserver {
```

Comment:
nit: `*Observer` isn't the right name here: that conflicts with the Observer pattern. Is there any reason `setVideos` can't be a method directly on the adapter? `adapter.setVideos` makes sense to me.